onnxruntime/core/optimizer/qdq_transformer/qdq_util.cc

```diff
@@ -240,34 +240,35 @@ bool GetDataTypeMinMax(int32_t data_type, int32_t& min, int32_t& max) {
   }
   return true;
 }
-bool GetQSalarScaleZp(const GraphViewer& graph_viewer, const Node& q_node, float& scale, int32_t& zp,
-                      int32_t& data_type) {
+bool GetQScalarScaleZp(const Graph& graph, const Node& q_node, float& scale, int32_t& zp, int32_t& data_type) {
```

Comment:
I actually wanted to change it to use GetInitializerFn to make it consistent with other functions in that file. But at the end I found that optimizer_utils::GetClipConstantMinMax requires at least a Graph, so I changed back to use Graph instead of Fn. Maybe using Graph can help for callers that have Graph but don't have GraphViewer?